### PR TITLE
(core) restrict load balancer tag instance counts to server group

### PR DESCRIPTION
**Pull request by anotherchrisberry**
The current rendering of the instance counts on the load balancer menu is a little misleading, as it shows the counts for all instances in the load balancer.

This change creates the instance counts from the instances in the server group, which is what users expect.

Also converting to a component and using Typescript, and removing the `maxDisplay` field, which was never used.

@zanthrash @icfantv PTAL

### CODE REVIEW

File: app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import {Application} from '../application/application.model.ts';
+
+export class LoadBalancersTagController implements ng.IComponentController {
+
+  public application: Application;
+  public serverGroup: any;
+  public loadBalancers: any[];
+
+  $onInit() {
```
Comment (icfantv) [inline app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts]: missing `public` keyword?  i know it's implied, but ng2 will require you explicitly state the access modifier.  i like this - it feels more java-esque.

File: app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+class LoadBalancersTagComponent implements ng.IComponentOptions {
+  public bindings: any = {
+    application: '=',
+    serverGroup: '='
+  };
+
+  public controller: ng.IComponentController = LoadBalancersTagController;
+  public templateUrl: string = require('./loadBalancer/loadBalancersTag.html');
+
+  constructor() {}
```
Comment (icfantv) [inline app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts]: Do we need an empty, no-arg ctor?

Comment (anotherchrisberry) [inline app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts]: Turns out we don't - I will remove, thanks for catching.

File: app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts (hunk 1 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts]: I can get behind that. Will update other TS files with explicit modifiers, too.

**Comment by icfantv** [comment]
LGTM

**Pull request merged**